Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM debian:buster-slim
+
+# Usage:
+# (1) Build image (only required once or after this file was changed):
+#     $ docker build -t photobooth .
+# (2) Start the container:
+#     $ docker run --name photobooth --rm -v "$(pwd):/var/www/html/" -p 8080:80 photobooth
+# (3) Go to: http://localhost:8080
+#
+# If you would like to connect to the running container:
+# $ docker exec -it photobooth /bin/bash
+
+RUN set -x \
+    && apt-get update \
+    && apt-get dist-upgrade -y \
+    && apt-get install -y nginx php-fpm php-gd gphoto2 unclutter
```

Comment:
unclutter is not needed here as we don't enable kiosk mode

Reply:
Right... removed it.